app/src/main/java/protect/card_locker/ManageGroupActivity.kt

```diff
@@ -34,37 +35,32 @@ class ManageGroupActivity : CatimaAppCompatActivity(), CardAdapterListener {
         super.onCreate(inputSavedInstanceState)
         binding = ActivityManageGroupBinding.inflate(layoutInflater)
         setContentView(binding.root)
-        Utils.applyWindowInsetsAndFabOffset(binding.getRoot(), binding.fabSave)
-        val toolbar: Toolbar = binding.toolbar
-        setSupportActionBar(toolbar)
+        Utils.applyWindowInsetsAndFabOffset(binding.root, binding.fabSave)
+        setSupportActionBar(binding.toolbar)
 
         mDatabase = DBHelper(this).writableDatabase
         noGroupCardsText = binding.include.noGroupCardsText
         mCardList = binding.include.list
 
-        val saveButton = binding.fabSave
         mGroupNameText = binding.editTextGroupName
-
         mGroupNameText.doAfterTextChanged {
             mGroupNameNotInUse = true
             mGroupNameText.error = null
             val currentGroupName = mGroupNameText.text.trim().toString()
             if (currentGroupName.isEmpty()) {
-                mGroupNameText.error = getResources().getText(R.string.group_name_is_empty)
+                mGroupNameText.error = getText(R.string.group_name_is_empty)
                 return@doAfterTextChanged
             }
             if (mGroup._id != currentGroupName) {
                 if (DBHelper.getGroup(mDatabase, currentGroupName) != null) {
                     mGroupNameNotInUse = false
-                    mGroupNameText.error =
-                        getResources().getText(R.string.group_name_already_in_use)
+                    mGroupNameText.error = getText(R.string.group_name_already_in_use)
                 } else {
                     mGroupNameNotInUse = true
                 }
             }
         }
 
-        val intent = getIntent()
```

Comment:
Is it really save to just remove this call?

Reply:
yes, we use kotlin val intent instead getIntent method